README.md

```diff
@@ -39,6 +37,85 @@ $ airflow connections add "wherobots_default" \
     --conn-password "$(< api.key)"
 ```
 
+## Usage
+
+### Execute a `Run` on Wherobots Cloud
+
+Wherobots allows users to upload their code (.py, .jar),
+run it on the cloud, and monitor the status of the run.
+
+The `WherobotsRunOperator` allows you to execute a `Run` on Wherobots Cloud.
+It will trigger the run according to the parameters you provide,
+and will wait for the run to finish before completing the task.
+
+Refer to the [Wherobots Managed Storage Documentation](https://docs.wherobots.com/latest/develop/storage-management/storage/)
+to learn more about how to upload and manage your code on Wherobots Cloud.
+
+Below is an example `WherobotsRunOperator`
+
+```python
+operator = WherobotsRunOperator(
+        task_id="your_task_id",
+        name="airflow_operator_test_run_{{ ts_nodash }}",
+        run_python={
+            "uri": "s3://wbts-wbc-m97rcg45xi/42ly7mi0p1/data/shared/classification.py"
+        },
+        runtime="tiny-a10-gpu",
+        dag=dag,
+        poll_logs=True,
+    )
+```
+
+#### Arguments
+
+The arguments for the `WherobotsRunOperator` constructor:
+
+The `run_*` arguments are mutually exclusive, you can only specify one of them.
+
+* `name: str`: The name of the run.
+  If not specified, a default name will be
+  generated.
+* `runtime: str`: The runtime decides the size of the workloads that execute the run.
+  The default value is `TINY`
+* `poll_logs: bool`: If `True`, the operator will poll the logs of the run until it finishes.
+  If `False`, the operator will not poll the logs, just track the status of the run.
+* `polling_interval`: The interval in seconds to poll the status of the run.
+  The default value is `30`.
+* `run_python: dict`: A dictionary with the following keys:
+  * `uri: str`: The URI of the Python file to run.
+  * `args: list[str]`: A list of arguments to pass to the Python file.
+* `run_jar: dict`: A dictionary with the following keys:
+  * `uri: str`: The URI of the JAR file to run.
+  * `args: list[str]`: A list of arguments to pass to the JAR file.
+  * `mainClass: str`: The main class to run in the JAR file.
+* `environment: dict`: A dictionary with the following keys:
+  * `sparkDriverDiskGB: int`: The disk size for the Spark driver.
+  * `sparkExecutorDiskGB: int`: The disk size for the Spark executor.
+  * `sparkConfigs: dict`: A dictionary of Spark configurations.
+  * `dependencies: list[dict]`: A list of dependant libraries to install.
+
+The `dependencies` argument is a list of dictionaries.
+There are two types of dependencies supported.
+
+1. `PYPI` dependencies:
+```json
+{
+    "sourceType": "PYPI",
+    "libraryName": "package_name",
+    "libraryVersion": "package_version"
+}
+```
+
+2. `FILE` dependencies:
+```json
+{
+    "sourceType": "FILE",
+    "filePath": "s3://bucket/path/to/dependency.whl"
+}
+```
+File types supported: `.whl`, `.zip`, `.jar`
+
+
 ### Execute a SQL query
```

Comment:
Would it be possible to have the same list of arguments for `WherobotsSqlOperator`?

Reply:
Sure I'll add separately